www/gatsby-node.js

```diff
@@ -291,6 +291,11 @@ exports.createPages = ({ graphql, actions, reporter }) => {
     toPath: `/tutorial/wordpress-source-plugin-tutorial/`,
     isPermanent: true,
   })
+  createRedirect({
+    fromPath: `/docs/prismic-source-graphql-tutorial/`,
```

Comment:
What is this redirect for? I'm not sure that we need it. There is a doc for sourcing from Prismic, which I think should stay where it is. If it has outdated content, the best thing would be to update that page to include less step-by-step guidance while still talking about how Gatsby Prismic can be sourced together (as an informational doc). Then, it can point to the advanced tutorial for guidance. https://www.gatsbyjs.org/docs/sourcing-from-prismic/

Reply:
Agreed, the existing guide is not outdated so it should remain. Removed the redirect

Reply:
The existing guide could be updated with the `prismic-cli` - I'd be happy to update it myself (as I wrote it) but I won't have time the next two weeks.